be/src/olap/rowset/column_data.cpp

```diff
@@ -138,13 +138,13 @@ OLAPStatus ColumnData::_seek_to_block(const RowBlockPosition& block_pos, bool wi
                               _lru_cache, _runtime_state, _stats);
         if (_segment_reader == nullptr) {
             OLAP_LOG_WARNING("fail to malloc segment reader.");
-            return OLAP_ERR_MALLOC_ERROR;
+            return Status::OLAPInternalError(OLAP_ERR_MALLOC_ERROR);
         }
 
         _current_segment = block_pos.segment;
         auto res = _segment_reader->init(_is_using_cache);
-        if (OLAP_SUCCESS != res) {
-            OLAP_LOG_WARNING("fail to init segment reader. [res=%d]", res);
+        if (!res.ok()) {
+            LOG(WARNING) << "fail to init segment reader. res = " << res;
```

Comment:
Why not using OLAP_LOG_WARNING?

Reply:
OLAP_LOG_XXXX will be replaced with LOG(WARNING) in the future. LOGGING SYSTEM should also be uniformed.